packages/create-redwood-app/template/web/jest.config.js

```diff
@@ -1 +1,6 @@
-module.exports = require('@redwoodjs/testing/config/jest/web')
+const config = {
```

Comment:
I suggest we add a link to Jest Project config section in redwoodjs.com/docs/testing section [TODO]

Reply:
👍